azerothcore/azerothcore-wotlk
# [Core/Scrips]: Kologarn focused eyebeam should target only 1 target.

## Changes Proposed:
-  Change eyebeam trigger MaxAffectedTargets to 1

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7351

## SOURCE:


## Tests Performed:

- Tested with the method below


## How to Test the Changes:


1. Login on more than 1 character
2. Make your way through ulduar to Kologarn
3. Engage the boss and wait 10 seconds for him to cast focused eyebeam
4. Observe if the beam now only targets one character

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
